packages/storybook/stories/Components/Pagination.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { PaginationItemsPerPage as ItemsPerPage, Pagination } from "../";
+import { action } from "@storybook/addon-actions";
+
+const meta = {
+  component: Pagination,
+  args: {
+    page: 1,
+    pageCount: 5,
+    itemsPerPage: 5,
```

Comment:
Here wrong values are correctly highlighted, but they do not cause any build error. Is that intended?